reviews-receiver/src/routes/batch.router.ts

```diff
@@ -60,4 +81,36 @@ const batchRouter = Router();
  */
 batchRouter.post('/', checkApiKey, schemaValidationMiddleware({ body: IBatchReviewList }), BatchController.post);
 
+/**
+ * @swagger
```

Comment:
why is this a different route? we can use the batch route that we already have

Reply:
I think that the separation of the routes make it easier to use:
batch route include: x-api-key header, businessId in the body
batch user interface include: JWT Token, without businessId in the body (extract from JWT verify func)